src/Control/Director.php

```diff
@@ -563,6 +563,11 @@ public static function is_https(HTTPRequest $request = null)
             }
         }
 
+        if (!empty($_SERVER['HTTPS']) && $_SERVER['HTTPS'] != 'off') {
```

Comment:
This code needs to be higher than `default_base_url` check, but lower than the currentRequest(), since `default_base_url` is almost never going to be empty, and will almost always be a poor approximation.